packages/fxa-settings/src/index.tsx

```diff
@@ -6,12 +6,13 @@ import React from 'react';
 import { render } from 'react-dom';
 import './index.scss';
 import App from './components/App';
+import sentryMetrics from '@fxa-shared/lib/sentry';
+import config from './lib/config';
 
 export async function init() {
-  // TODO: Configure Sentry logging
-  // if (config.sentry.dsn) {
-  //   sentryMetrics.configure(config.sentry.dsn, config.version);
-  // }
+  if (config.sentry.dsn) {
```

Comment:
I'm not sure if there's a downside to this, but I noticed in content-server we get the error `No Sentry dsn provided` logged unnecessarily.

Reply:
If there's no `config.sentry.dsn` then it's likely there's no `config.sentry`.  Check for that?

I actually like that message, so we know there's an issue.  We could add an 'enable' flag to explicitly disable Sentry.

Reply:
> I actually like that message, so we know there's an issue.

Is it ever active in local development?

Reply:
> If there's no config.sentry.dsn then it's likely there's no config.sentry.

I think `config.sentry` will always exist, but our default is an [empty string](https://github.com/mozilla/fxa/pull/5203/files#diff-6f6012abb4afb051e7e0b053b896fedaR15), which means if we keeps this I should check for string length instead I guess

Reply:
> 
> Is it ever active in local development?

No, unless someone is working on Sentry related stuff.  But we did see it on prod Payments and realized the dsn is missing there.

Reply:
Fair point!